pkg/config/kvclient.go

```diff
@@ -22,6 +22,8 @@ type KVClientConfig struct {
 	WorkerConcurrent uint `toml:"worker-concurrent" json:"worker-concurrent"`
 	// how many grpc streams will be established to every TiKV node
 	GrpcStreamConcurrent uint `toml:"grpc-stream-concurrent" json:"grpc-stream-concurrent"`
+	// Advance table ResolvedTs interval.
+	AdvanceIntervalInMs uint `toml:"advance-interval-in-ms" json:"advance-interval-in-ms"`
```

Comment:
Would you think `WatermarkAdvanceIntervalInMs` is a better name?

Reply:
I prefer to a shorter name. Besides `ResolvedTs`, there is no other things can be advanced in puller. So I think the short name is OK.